### PR TITLE
updates and version increments v0.13.4

### PR DESCRIPTION
**Pull request by sjhx**
addressing vulnerabilities
and https://github.com/IBM/portieris/issues/433

### CODE REVIEW

**Review by mikeyoung3k — APPROVED** [review]
LGTM

File: scripts/makeKeySecret.sh [hunk 1]
```diff
@@ -9,6 +9,10 @@ echo 'where <key-id> can be obtained from `gpg -k --key-id-format=long`'
 fi
 shift
 
+if [ "$*" == "" ]; then
```
Comment (tim-gp) [inline scripts/makeKeySecret.sh:12]: what's this?

Comment (sjhx) [reply scripts/makeKeySecret.sh:12]: its a minor enhancement sliding in under the covers 
it reminds a consumer of this totally optional, just for help, script that if they dont say what key they want they get them all

**Review by tim-gp — APPROVED** [review]
LG

**Pull request merged**